spec/services/dttp/account_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "rails_helper"
```

Comment:
Can we add a whitespace line to be consistent?

Reply:
done